services/opaque.go

```diff
@@ -21,10 +21,6 @@ const (
 	opaqueSecretKeyEnv  = "OPAQUE_SECRET_KEY"
 	opaquePublicKeyEnv  = "OPAQUE_PUBLIC_KEY"
 	opaqueFakeRecordEnv = "OPAQUE_FAKE_RECORD"
-
-	opaqueArgon2TimeParam     = 2
```

Comment:
Are these the default now?

Reply:
It turns out that the KSF is only used on the client side, so these parameters are irrelevant for the server.